server/index.js

```diff
@@ -172,6 +173,24 @@ class Server extends BaseServer {
       return process.exit(1)
     }
   }
+
+  async autoloadModules (pattern = 'modules/**/*.js') {
```

Comment:
The pattern is not correct.

It will load `modules/users/index.test.js` or `modules/users/lib/utils.js`, but will not load `modules/users.js`

Reply:
Let’s add more file to ` test/servers/modules/`